rotkehlchen/chain/ethereum/modules/aave/v2/decoder.py

```diff
@@ -183,6 +184,43 @@ def _decode_borrow(
                 # Set protocol for both events
                 event.counterparty = CPT_AAVE_V2
 
+    def _decode_liquidation(
+            self,
+            tx_log: EvmTxReceiptLog,
+            decoded_events: list['EvmEvent'],
+    ) -> None:
+        """
+        Decode AAVE v2 liquidations. When a liquidation happens the user returns the debt token
+        and part of the collateral deposited is lost too. Those two events happen as transfers in
+        a transaction started by the liquidator.
+        """
+        if self.base.is_tracked(hex_or_bytes_to_address(tx_log.topics[3])) is False:
+            return
+
+        for event in decoded_events:
+            if event.event_type != HistoryEventType.SPEND:
+                continue
+
+            asset = event.asset.resolve_to_evm_token()
+            if asset_normalized_value(
+                amount=hex_or_bytes_to_int(tx_log.data[32:64]),  # liquidated amount
+                asset=asset,
+            ) == event.balance.amount and asset.protocol == CPT_AAVE_V2:
+                # we are transfering the aTOKEN
+                event.event_subtype = HistoryEventSubType.LIQUIDATE
+                event.notes = f'{event.balance.amount} {asset.symbol} got liquidated'
+                event.counterparty = CPT_AAVE_V2
+                event.address = tx_log.address
+            elif asset_normalized_value(
+                amount=hex_or_bytes_to_int(tx_log.data[:32]),  # debt amount
+                asset=asset,
+            ) == event.balance.amount:
+                # we are transfering the debt token
+                event.event_subtype = HistoryEventSubType.PAYBACK_DEBT
+                event.notes = f'Payback {event.balance.amount} {asset.symbol}'
```

Comment:
What exactly does this event represent?

```suggestion
                event.notes = f'Payback {event.balance.amount} {asset.symbol} for an aave-v2 position'
```